site-scrapers/TuftsMC/config.js

```diff
@@ -0,0 +1,47 @@
+const site = {
+    name: "Tufts Medical Center",
+    street: "279 Tremont St",
+    city: "Boston",
+    zip: "02116",
+    signUpLink:
+        "https://www.zocdoc.com/wl/tuftscovid19vaccination/practice/64825?reason_visit=5243",
+};
+
+const requestInit = {
+    headers: {
+        accept: "*/*",
+        "accept-language": "en-US,en;q=0.9",
+        "content-type": "application/json",
+        "sec-ch-ua":
+            '"Google Chrome";v="89", "Chromium";v="89", ";Not A Brand";v="99"',
+        "sec-ch-ua-mobile": "?0",
+        "sec-fetch-dest": "empty",
+        "sec-fetch-mode": "cors",
+        "sec-fetch-site": "same-site",
+        "x-zd-application": "patient-web-app",
+        "x-zd-referer": "",
+        "x-zd-url":
+            "https://www.zocdoc.com/wl/tuftscovid19vaccination/practice/64825?reason_visit=5243",
+        "x-zdata": "eyJob3N0Ijoid3d3LnpvY2RvYy5jb20ifQ==",
+        "zd-application-name": "patient-web-app",
+        "zd-pageview-id": "205cc2a39612c66a00163be877d1b493",
+        "zd-referer": "",
+        "zd-session-id": "d61cc2b74bd246029b988a4038123001",
+        "zd-tracking-id": "b28be354-5d7f-4ea2-bdad-1e05b17554d0",
+        "zd-url":
+            "https://www.zocdoc.com/wl/tuftscovid19vaccination/practice/64825?reason_visit=5243",
```

Comment:
you're repeating this website several times. Mind referring to it as site.signUpLink in all cases maybe?

Reply:
Three things: 1) with the intended inclusion of Tufts MC into the more encompassing ZocDoc scraper, this goes away. 2) That was an incomplete refactoring where I'd remove the in-line string in the function that uses it so as to make the code in the scraper less cluttered. 3) The content of `requestInit` is copied from Chrome's inspector, and I didn't want to tinker with it because my head starts to spin just looking at it, and the "danger, danger, Will Robinson" voice goes off in my head: Leave well enough alone; don't fix (improve) it if it isn't broken.